notebooks/226-whisper-subtitles-generation/226-whisper-subtitles-generation.ipynb

```diff
@@ -0,0 +1,900 @@
+{
+ "cells": [
```

Comment:
I am ok with the code, a lot of work has been done. But, from my perspective, the value of such a notebook is low mostly due to the following reason. The notebook just demonstrates the capability to bring the Whisper model to OpenVINO and infer it but the reuse is limited by the complexity of the code. To me, the best way to overcome such an issue is to follow the community trend when the complicated pipelines are wrapped with simple API. The best example here is HF. I think it could be reasonable to have a high-level wrapper for this model in the OpenVINO Model API, use it in this notebook and let the community try it easily. The API could look like as follows:
```python
text = model.transcribe(audio, task) # the rest of the parameters are default
```

Reply:
so I suggest to discuss offline how we will move in this direction
cc @Wovchena